libvips/colour/LCh2UCS.c

```diff
@@ -94,7 +94,7 @@ vips_col_L2Lcmc( float L )
 	if( L < 16.0 )
 		Lcmc = 1.744 * L;
 	else
-		Lcmc = 21.75 * log( L ) + 0.3838 * L - 38.54;
+		Lcmc = 21.75 * logf( L ) + 0.3838 * L - 38.54;
```

Comment:
logf() and friends are a C99 feature and many older compilers don't implement C99 fully. I think we'd need some tests in configure for this commit. For example, we test for inline (another C99 feature).